web/i18n/en-US/workflow.ts

```diff
@@ -74,6 +74,8 @@ const translation = {
     exportPNG: 'Export as PNG',
     exportJPEG: 'Export as JPEG',
     exportSVG: 'Export as SVG',
+    currentView: 'Current View',
+    wholeWorkflow: 'Whole Workflow',
```

Comment:
The functions works well, however I do suggest changing the `wholeWorkflow` to `currentWorkflow` instead.
```suggestion
    currentWorkflow: 'Current Workflow',
```